src/sonic-yang-models/yang-templates/sonic-types.yang.j2

```diff
@@ -367,4 +367,14 @@ module sonic-types {
     }
     {% else %}
     {% endif %}
+
+    typedef path_tracing_timestamp_template {
```

Comment:
Could you add some additional details about templates 1-4 in this model description?

Reply:
Hi @kperumalbfn Many thanks for your review. I added the templates description.